### ISSUE TITLE
MPI_Status is declared in the Fortran modules

### ISSUE THREAD

**Issue by jeffhammond** [mpi-forum/mpi-issues#659]
# Problem

The following does not mention the Fortran modules.  It should have.

> 3.2.6 Passing MPI_STATUS_IGNORE for Status
> Every call to MPI_RECV includes a status argument, wherein the system can return details about the message received. There are also a number of other MPI calls where status is returned. An object of type MPI_Status is not an MPI opaque object; its structure is declared in mpi.h and mpif.h, and it exists in the user’s program. In many cases, application programs are constructed so that it is unnecessary for them to examine the status fields. In these cases, it is a waste for the user to allocate a status object, and it is particularly wasteful for the MPI implementation to fill in fields in this object.

# Proposal

Add Fortran modules to "its structure is declared in mpi.h and mpif.h,"

# Changes to the Text

Add Fortran modules to "its structure is declared in mpi.h and mpif.h,"

# Impact on Implementations

None

# Impact on Users

Clarity in the standard.

# References and Pull Requests

**Comment by wesbland** [comment]
Is this something intended for MPI 4.1? If so, it needs to be read at the July meeting and announced by June 26th.

**Comment by jeffhammond** [comment]
This seems like a chapter committee change, or an editorial one, and not a high priority in either way, because even though there's an omission here, absolutely no one has struggled to use MPI because of it.

**Comment by wgropp** [comment]
I agree that it is a chapter committee change. This is also another example of the folly of enumerating instances, in this case, mpi.h and mpif.h. Better would be a term for all of these, though we don't have one.

**Comment by RolfRabenseifner** [comment]
Yes, "in mpi.h and mpif.h," on page 41 line 24 should be substituted by, e.g., "in mpi.h, the Fortran mpi_f08 and mpi modules, and mpif.h,".
